test/integration/ratelimit_integration_test.cc

```diff
@@ -162,13 +162,13 @@ class RatelimitIntegrationTest : public HttpIntegrationTest,
 
   const uint64_t request_size_ = 1024;
   const uint64_t response_size_ = 512;
-  bool failure_mode_ = false;
+  bool failure_mode_ = true;
```

Comment:
nit: sorry I don't think I was clear with my previous comment. Now that you changed the logic of this var, can we change the name to match the config name `failure_mode_allow`. I think that adds clarity.